tests/unit/test_store.py

```diff
@@ -17,9 +17,15 @@
 
 class TestStoreSessions(unittest.TestCase):
     @responses.activate
```

Comment:
Should we have a test that test both token and EDL credentials are present? other than that I think the code change in auth addresses the core issue.